src/kili/llm/services/export/dynamic.py

```diff
@@ -168,6 +172,7 @@ def _build_rounds(self, chat_items, annotations, json_interface):
                         annotation
                         for annotation in annotations
                         if annotation["chatItemId"] == chat_item["id"]
+                        or annotation["chatItemId"] is None
```

Comment:
Not sure we should add this condition here.

If we do so, for each assistant response we will add all annotations without chatItemId.
If our asset haven't any assistant response, annotation whithout chat item id won't be added.

For me we should do it separately, just before appending the current_round to rounds.